libs/docs/src/docs/2.2.1-alpha/overriding-pod-and-container-attributes.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Extend Kubernetes resources for container components
+description: Extend Kubernetes resources for container components through attributes
```

Comment:
```suggestion
description: Extending kubernetes resources
```

We can also rename this file to `extending-kubernetes-resources` since we don't have a section just for kubernetes resources